PIVX-Labs/MyPIVXWallet
# [TEST] Add unit tests for GenKeyWarning

## Abstract

Add unit tests for the GenKeyWarning class, I have also found a bug since we were trying to change a props, which is readonly. The event `open` has been added in place.
Tests do the following:

- Check that all input boxes/buttons are visible only when they should be
- Check that events are emitted as expected with the expected amount of arguments
- Check that errors are throwed as expected (this has been done by mocking alerts and createalert)
